Classes/Controller/NavigationController.php

```diff
@@ -25,6 +25,28 @@
  */
 class NavigationController extends AbstractController
 {
+    /**
+     * Method to get the page select values and use them with chash
+     * @param \Kitodo\Dlf\Domain\Model\PageSelectForm|NULL $pageSelectForm
+     * @return void
+     */
+    public function pageSelectAction(\Kitodo\Dlf\Domain\Model\PageSelectForm $pageSelectForm = NULL) {
+        if ($pageSelectForm) {
+            $uriBuilder = $this->getControllerContext()->getUriBuilder();
+            $uri = $uriBuilder->reset()
+                ->setArguments(
+                    [
+                        'tx_dlf' => [
+                            'id' => $pageSelectForm->getId(),
+                            'page' => $pageSelectForm->getPage()
```

Comment:
Do I miss something or did you miss the `tx_dlf[double]` parameter?

Reply:
Nope, you are right!
Should be fixed now.